Test Case 1 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 2 Input:
30 1 52 0 31 2 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 3 Input:
30 1 21 2 30 2 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 4 Input:
11 2 3
Output:
No
Test Case 5 Input:
30 1 21 2 32 0 421 2 12 0 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 6 Input:
30 1 21 2 32 0 420 1 51 2 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 7 Input:
11 2 5
Output:
No
Test Case 8 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 9 Input:
30 1 51 2 32 0 8
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 10 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 11 Input:
30 1 51 2 32 0 "2"
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 12 Input:
50 1 51 2 32 0 23 2 44 1 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 13 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 14 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 15 Input:
20 1 31 2 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 16 Input:
10 1 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 17 Input:
10 1 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 18 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 19 Input:
20 1 31 2 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 20 Input:
00 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 21 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 22 Input:
10 1 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 23 Input:
30 1 51 2 32 0 22 0 2Note: This test case is the same as the first test case in the previously generated test cases.
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 24 Input:
-20 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:12)
Test Case 25 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 26 Input:
10 1 51 2 32 0 23
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 27 Input:
10 1 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 28 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 29 Input:
41 2 32 3 43 4 54 5 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
	at Main.main(Main.java:15)
Test Case 30 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 31 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 32 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 33 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 34 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 35 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 36 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 37 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 38 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 39 Input:
20 1 101 2 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 40 Input:
Test Case Input:0Test Case Input:30 1 51 2 32 0 2Test Case Input:30 1 51 2 32 0 2Test Case Input:40 1 21 2 32 3 43 0 5Test Case Input:20 1 51 2 3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 41 Input:
0Test Case Input:30 1 51 2 32 0 2Test Case Input:20 1 51 2 3Test Case Input:40 1 21 2 32 3 43 0 5
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 42 Input:
0Test Case Input:30 1 51 2 32 0 2Test Case Input:-21 2 33 4 56 7 8Test Case Input:"abc"Test Case Input:50 1 2 3 41 2 3 4 52 3 4 5 63 4 5 6 74 5 6 7 8
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 43 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 44 Input:
Test Case Input:
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 45 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 46 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 47 Input:
Test Case Input:30 1 51 2 32 0 2Test Case Input:0
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 48 Input:
Test Case Input:Test Case Input:Test Case Input:0Test Case Input:30 1 51 2 32 0 2Test Case Input:30 1 51 2 32 0 2Test Case Input:40 1 21 2 32 3 43 0 5Test Case Input:20 1 51 2 3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 49 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 50 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 51 Input:
10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 52 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 53 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 54 Input:
Test Case Input:Test Case Input:Test Case Input:Test Case Input:Test Case Input:Test Case Input:Test Case Input:Test Case Input:Test Case Input:
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 55 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 56 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 57 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 58 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 59 Input:
Test Case Input: 30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 60 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 61 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 62 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 63 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 64 Input:
10 1 51 2 32 0 23
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 65 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 66 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 67 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 68 Input:
Test Case Input:30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 69 Input:
Test Case Input:30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 70 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 71 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 72 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 73 Input:
Test Case Input:30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 74 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 75 Input:
Test Case Input:
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 76 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 77 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 78 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 79 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 80 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 81 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 82 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 83 Input:
-20 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:12)
Test Case 84 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 85 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 86 Input:
-0 1 51 2 32 0 23 0 2Test Case Input:null0 1 51 2 32 0 23 0 2Test Case Input:70 1 -51 2 32 0 23 0 2Test Case Input:30 1 51 2 32 0 23 0 -2Test Case Input:20 1 51 2 32 0 23 0 2Test Case Input:30 1 51 2 32 0 23 0 2Test Case Input:40 1 21 2 32 3 43 0 5Test Case Input:20 1 51 2 3Test Case Input:a0 1 51 2 32 0 23 0 2Test Case Input:null0 1 51 2 32 0 23 0 2Test Case Input:30 1 51 2 32 0 23 0 2Test Case Input:-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "-"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 87 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 88 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 89 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 90 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 91 Input:
11 2 3
Output:
No
Test Case 92 Input:
1a b c0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 93 Input:
0Test Case Input:30 1 51 2 32 0 2Test Case Input:40 1 21 2 32 3 43 0 5Test Case Input:20 1 51 2 3Test Case Input:a0 1 51 2 32 0 2Test Case Input:-10 1 51 2 32 0 2Test Case Input:a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 94 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 95 Input:
-0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "-"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 96 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 97 Input:
10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 98 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 99 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 100 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 101 Input:
Test Case Input: 30 1 51 2 32 0 2Test Case Input:
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 102 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 103 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 104 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 105 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 106 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 107 Input:
0
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.trim()" because the return value of "java.io.BufferedReader.readLine()" is null
	at Main.next(Main.java:67)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 108 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 109 Input:
00 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 110 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 111 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 112 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 113 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 114 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 115 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 116 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 117 Input:
00 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 118 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 119 Input:
00 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 120 Input:
1a 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 121 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 122 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 123 Input:
Test Case Input:30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 124 Input:
20 11 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 125 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 126 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 127 Input:
0Test Case Input:20 1 51 2 3Test Case Input:30 1 51 2 32 0 2Test Case Input:a0 1 51 2 32 0 2Test Case Input:1a 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 128 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 129 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 130 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 131 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 132 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 133 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 134 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 135 Input:
3a b cd e fg h i
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 136 Input:
10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 137 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 138 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 139 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 140 Input:
20 1 51 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 141 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 142 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 143 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 144 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 145 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 146 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 147 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 148 Input:
Test Case Input:
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 149 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 150 Input:
40 1 21 2 32 3 43 0 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 151 Input:
-10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 152 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 153 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 154 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 155 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 156 Input:
00 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 157 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 158 Input:
a0 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 159 Input:
30 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 160 Input:
Test Case Input:10 1 51 2 32 0 2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "Test"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 161 Input:
11
Output:
No
Test Case 162 Input:
11
Output:
No
Test Case 163 Input:
0 6
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.trim()" because the return value of "java.io.BufferedReader.readLine()" is null
	at Main.next(Main.java:67)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:18)
Test Case 164 Input:
11
Output:
No
Test Case 165 Input:
11
Output:
No
Test Case 166 Input:
"11111"5
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""11111""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 167 Input:
11
Output:
No
Test Case 168 Input:
01
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.size()" because "<parameter1>[<local3>]" is null
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 169 Input:
"12345"6
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""12345""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 170 Input:
11
Output:
No
Test Case 171 Input:
"123"3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""123""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 172 Input:
"1234"5
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""1234""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 173 Input:
"123"5
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""123""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 174 Input:
"abc"3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""abc""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 175 Input:
-50
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 176 Input:
"123"4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""123""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 177 Input:
"abc"3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""abc""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 178 Input:
"abc"3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""abc""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 179 Input:
"Hello"2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""Hello""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 180 Input:
"abc"2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""abc""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 181 Input:
-13
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.trim()" because the return value of "java.io.BufferedReader.readLine()" is null
	at Main.next(Main.java:67)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:18)
Test Case 182 Input:
-12
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.trim()" because the return value of "java.io.BufferedReader.readLine()" is null
	at Main.next(Main.java:67)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:18)
Test Case 183 Input:
abc0
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "abc"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 184 Input:
null5
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 185 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 186 Input:
abc3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "abc"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 187 Input:
a0
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 188 Input:
-23
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:12)
Test Case 189 Input:
-52
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 190 Input:
-52
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 191 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 192 Input:
1234
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
	at Main.main(Main.java:15)
Test Case 193 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 194 Input:
-51
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 195 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 196 Input:
abc0
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "abc"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 197 Input:
"abc"2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""abc""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 198 Input:
null3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 199 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 200 Input:
1abc
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "abc"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 201 Input:
-51
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 202 Input:
-51
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 203 Input:
1-5
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:13)
Test Case 204 Input:
-52
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 205 Input:
2abc
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "abc"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:11)
Test Case 206 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 207 Input:
-53
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 208 Input:
null1
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 209 Input:
null2
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "null"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 210 Input:
10
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 211 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 212 Input:
432 31 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 213 Input:
321 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 214 Input:
431 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 215 Input:
541 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
	at Main.main(Main.java:15)
Test Case 216 Input:
541 21 32 43 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
	at Main.main(Main.java:15)
Test Case 217 Input:
541 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
	at Main.main(Main.java:15)
Test Case 218 Input:
321 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 219 Input:
321 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 220 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 221 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 222 Input:
4 31 22 33 45 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 223 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 224 Input:
0 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 225 Input:
a 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 226 Input:
4 01 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 227 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 228 Input:
4 31 02 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 229 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 230 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 231 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 232 Input:
2 11 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
	at Main.main(Main.java:15)
Test Case 233 Input:
4 21 12 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 234 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 235 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 236 Input:
4 31 22 33 45Test Case Input:4 01 22 33 4Test Case Input:4 31 22 33 4Test Case Input:4 01 22 33 4Test Case Input:4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 237 Input:
4 31 22 33 45 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 238 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 239 Input:
4 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 240 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 241 Input:
-5 31 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 242 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 243 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 244 Input:
4 31 22 33 4Test Case Input:0 01 22 33 4Test Case Input:4 01 22 33 4Test Case Input:4 01 22 33 4Test Case Input:4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 245 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 246 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 247 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 248 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 249 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 250 Input:
5 51 21 32 43 44 5
Output:
No
Test Case 251 Input:
0 21 22 3
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 252 Input:
1 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 253 Input:
0 01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 254 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 255 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 256 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 257 Input:
0 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 258 Input:
0 41 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 259 Input:
5 41 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
	at Main.main(Main.java:15)
Test Case 260 Input:
0 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 261 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 262 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 263 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 264 Input:
a 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 265 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 266 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 267 Input:
-2 31 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:12)
Test Case 268 Input:
1 01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 269 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 270 Input:
2 21 22 1
Output:
Yes
1
Test Case 271 Input:
2 21 22 1
Output:
Yes
1
Test Case 272 Input:
2.5 3.71.2 3.45.6 7.89.0 10.1
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "2.5"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 273 Input:
4 31 22 33 null
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 274 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 275 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 276 Input:
-1 51 22 33 44 5
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 277 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 278 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 279 Input:
2 21 22 1
Output:
Yes
1
Test Case 280 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 281 Input:
-4 31 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -3
	at Main.main(Main.java:12)
Test Case 282 Input:
0 21 23 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 283 Input:
4 21 22 34 5Explanation: This test case has a different number of edges (M) than the number of given edges in the input.
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 284 Input:
5 01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 285 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 286 Input:
2.5 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "2.5"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 287 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 288 Input:
2 11 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
	at Main.main(Main.java:15)
Test Case 289 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 290 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 291 Input:
2 01 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 292 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 293 Input:
0 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 294 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 295 Input:
0 21 22 3
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 296 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 297 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 298 Input:
4 21 23 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 299 Input:
1 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 300 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 301 Input:
4 21 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 302 Input:
2 21 22 0
Output:
Yes
1
Test Case 303 Input:
```4 31 22 33 4```Test Case Input:```4 01 22 33 4```Test Case Input:```4 01 22 33 4```Test Case Input:```5 51 21 32 43 44 5```Test Case Input:```0 0```Test Case Input:```2 21 22 1```Test Case Input:```4 31 22 33 4```Test Case Input:```4 31 22 33 4```Test Case Input:```-3 01 22 33 4```
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "```"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 304 Input:
"4" "3""1" "2""2" "3""3" "4"
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""4""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 305 Input:
2 21 22 1
Output:
Yes
1
Test Case 306 Input:
2 11 2
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
	at Main.main(Main.java:15)
Test Case 307 Input:
-4 31 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -3
	at Main.main(Main.java:12)
Test Case 308 Input:
5 01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 309 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 310 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 311 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 312 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 313 Input:
a 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 314 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 315 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 316 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 317 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 318 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 319 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 320 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 321 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 322 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 323 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 324 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 325 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 326 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 327 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 328 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 329 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 330 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 331 Input:
0 0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.bfs(Main.java:46)
	at Main.main(Main.java:23)
Test Case 332 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 333 Input:
-5 31 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:12)
Test Case 334 Input:
4 31 a2 34 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 335 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 336 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 337 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 338 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 339 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 340 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 341 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 342 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 343 Input:
5 01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 344 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 345 Input:
3 31 22 33 1
Output:
Yes
1
2
Test Case 346 Input:
0 31 22 3
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 347 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 348 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 349 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 350 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 351 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 352 Input:
2 21 a2 3
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:19)
Test Case 353 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 354 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 355 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 356 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 357 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 358 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 359 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 360 Input:
7 61 21 32 43 44 54 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 7 out of bounds for length 7
	at Main.main(Main.java:15)
Test Case 361 Input:
2 21 22 1
Output:
Yes
1
Test Case 362 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 363 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 364 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 365 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 366 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 367 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 368 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 369 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 370 Input:
3 21 22 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:15)
Test Case 371 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 372 Input:
-3 21 22 3
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 373 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 374 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 375 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 376 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 377 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 378 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 379 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 380 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 381 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 382 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 383 Input:
5 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 384 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 385 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 386 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 387 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 388 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 389 Input:
4 01 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 390 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 391 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 392 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 393 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 394 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 395 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 396 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 397 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 398 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 399 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 400 Input:
"a" 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: ""a""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:648)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 401 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 402 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 403 Input:
4 31 22 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 404 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 405 Input:
a 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 406 Input:
4 31 a2 33 4
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:15)
Test Case 407 Input:
0 31 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.add(Object)" because "<local3>[<local5>]" is null
	at Main.main(Main.java:20)
Test Case 408 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
Test Case 409 Input:
a 31 22 33 4
Error:
Exception in thread "main" java.lang.NumberFormatException: For input string: "a"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at java.base/java.lang.Integer.parseInt(Integer.java:778)
	at Main.nextInt(Main.java:57)
	at Main.main(Main.java:10)
Test Case 410 Input:
-3 01 22 33 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:12)
